packages/react-i18n/src/i18n.ts

```diff
@@ -474,7 +474,7 @@ export class I18n {
     }).toLocaleLowerCase();
 
     if (isToday(date)) {
-      return time;
+      return `Today at ${time}`;
```

Comment:
We should not hard code the string here. We should use the translation file to translate this into multiple languages. For example, in French, the text should be `Aujourd'hui à ${time}`. 

Updating this is considered a breaking change because the consumer needs to add a new translation key. I'm wondering if there is a reason why we didn't add this version in the first place. Maybe @TheMallen has more context since he works more than me on this package.